### DISCUSSION TITLE
✨Help Prioritise the D&I Workgroup’s Efforts

### DISCUSSION THREAD

**Discussion by sayantikabanik** [psf/diversity-and-inclusion-wg#10] [Ideas]
Hi Everyone! 👋

The PSF Diversity and Inclusion Workgroup always seeks to learn from the community and improve its impact. As we plan for the future, we’d love to hear from you!

We want to open this discussion to brainstorm focus areas, initiatives, or ideas that the workgroup can prioritize in the upcoming months. This is a place to share your thoughts, observations, and suggestions for where we should direct our attention.

`To guide the discussion, here are some questions to consider:`

-  What areas within the Python community would benefit most from focused D&I efforts?
(e.g., specific regions, underrepresented groups, accessibility needs, or overlooked areas)

- Are there gaps in resources, events, or support that the D&I Workgroup could address?

- What are some initiatives (big or small) that you feel would have a measurable impact?
(Examples: mentorship programs, workshops, outreach campaigns, collaborations)

-  Are there any existing efforts or best practices from other organizations that could inspire our work?

-  How can we make it easier for community members to participate in D&I discussions and actions?

Feel free to respond to the above thoughts or share beyond the guided template. Whether it's a new idea, feedback on past initiatives, or something you’ve noticed that needs attention, we’re all ears!

**Comment by georgically** [comment]
Here's the link to our previous survey forms about D&I Awareness created by @iqbalabd previously:
[https://forms.gle/KNQkrXJHeMeScMm37](https://forms.gle/KNQkrXJHeMeScMm37
)
